quay/quay-operator
# ClusterServiceVersion

### Description

Add initial `ClusterServiceVersion` for install using Operator Lifecycle Manager.

### Testing

1. Create the `CatalogSource` in your cluster to make the Operator package available:
```sh
$ kubectl create -n openshift-marketplace -f ./deploy/quay-operator.catalogsource.yaml
```

2. Create the `Subscription` to tell OLM to install the Operator in your namespace (can also be done via *OperatorHub* UI):
```sh
$ kubectl create -n <your-namespace> -f ./deploy/quay-operator.subscription.yaml
```

3. Confirm Operator is running in UI and via CLI:
```sh
$ kubectl get clusterserviceversions -n <your-namespace>
```

### Screenshots

**Operator detail view:**
![Screenshot_20200720_115654](https://user-images.githubusercontent.com/11700385/87975203-298db980-ca80-11ea-9bd6-966f8dbbc2d3.png)

Addresses https://issues.redhat.com/browse/PROJQUAY-885
